src/server/app.js

```diff
@@ -65,13 +61,13 @@ Promise.config({
 const app = express();
 
 // Set up jade as view engine
-app.set('views', path.join(__dirname, 'templates'));
+app.set('views', path.join(__dirname, '../../templates'));
```

Comment:
It feels a little bad to be making this so relative, but could we mitigate by having a single `rootPath` const that encapsulates the `__dirname` and `../../` parts, then join that with the relative paths for the other bits?